docs/quality-assurance-and-test.md

```diff
@@ -4,43 +4,147 @@ Welcome to the introduction on how we are working with Quality Assurance and Tes
+Building a product always gives reason to consider product risk areas and the possible costs of these risks elements. In this section we would like to elaborate on how we are working with these considerations, and give inspration on how to practically address this.
+
+In a project like this product risk contains two overall questions
+
+1. *Are we building the right product?*
+2. *Are we building the product right?*
+
+In regards to the first question - we want to be certain that the solution is delivering us the value we need. This question can be answered by evaulating on the quality characteristics of the product. Do the functionality reach the expected level of functionality, security, performance, maintainability, usability etc. Here it is important to have all the relevant considerations outlined before starting to implement. What is the value, the targets, the expectations and the formal requirements for this product.
+
+Writing acceptance criteria and/or use case scenarios before starting to develop is a great way to ensure that all considerations are taken into account during the development phase. But don't forget to try and reason for the costs of not delivering to target or expectation.
+
+Building the right product has in earlier times been proven by a post-developing test - We want to have this mitigation before implementing, as a way of ensuring alignment of expectations. Thus the knowlegde and the learnings of this risk evaluation could also be used for automating regression test within the pipeline so we remember to re-visit our acceptance criteria throughout the delivery pipeline.
+
+Regarding the second question on how we are delevoping our solutions. Here we talk about issues that could affect the way we are able to deliver a solution. Considering complexity, knowledge levels, tools, experience etc. is important, as these risks would very easily impact the work needed and the quality of the solution.
+
+*Are you working on a product/solution and are you unsure of the risks - or how to mitigate certain risks you have identified? - use the community for collaboration and discussion.*
+
+### Working with risk
+
+One way of mitigating risks is testing - but we are all familiar with the old saying that test is a timeboxed event/a bottleneck before go live - In Green Energy we wish to ensure that test/QA is happening all the time, by everyone. We don't have a verification team - we are all responsible for mitigating risks, and ensuring quality - so rest assure that it is never too early to talk quality.
 
 ### Testing Quadrants
 
-A way of mitigating risk factors.
+One way of translating risk items to actual testing could be using the testing quadrants for identifying - the relevant test types for specific areas.
```

Comment:
For people not used to working with risk and test, this could be a way of identifying risks and how to test them - we cannot by default expect people to be able to draw the picture of their test sceanrios and what kind of test needs to be done.